Cargo.toml

```diff
@@ -15,9 +15,14 @@ repository = "https://github.com/ebfull/pairing"
 rand = "0.4"
 byteorder = "1"
 clippy = { version = "0.0.200", optional = true }
+serde = { version = "1.0.66", optional = true }
 
 [features]
 unstable-features = ["expose-arith"]
 expose-arith = []
+serialization-serde = ["serde"]
```

Comment:
Can the feature be called `serde` without violating recommended practices?

Reply:
Then Cargo complains:

```
Features and dependencies cannot have the same name: `serde`
```

Maybe `use-serde`? `serde-support`? Not sure whether those are much better.

Reply:
You don't need to have an explicit feature; when a crate is declared as an optional dependency it becomes an implicit feature: https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section